src/AppBundle/Form/PostType.php

```diff
@@ -49,13 +49,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label' => 'label.summary',
             ])
             ->add('content', null, [
-                'attr' => ['rows' => 20],
+                'attr' => [
+                    'rows' => 20,
+                    'help' => 'Markdown is allowed.',
```

Comment:
should it realy be added as a ``help`` attribute ? This is not valid HTML.

If you plan to display it in JS, you should use ``data-help``.
If you plan to display it in Twig, you should use a form type extension to add a new option (and then a new variable in FormView based on it).

Btw, the help message should be translated too